.circleci/config.yml

```diff
@@ -1,49 +1,54 @@
-version: 2
 jobs:
-  lint:
+  lighthouse:
     docker:
-      - image: circleci/node:9.11.2
-
+      - image: 'circleci/node:9.8.0-browsers'
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - 'v1-dependencies-{{ checksum "package.json" }}'
+            - v1-dependencies-
+      - run:
+          command: 'npm run build:app'
+          name: Start
+      - run:
+          command: 'npm run test:lighthouse'
+          name: Lighthouse
     working_directory: ~/repo
-
+  lint:
+    docker:
+      - image: 'circleci/node:9.11.2'
     steps:
       - checkout
-
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package.json" }}
-            # fallback to using the latest cache if no exact match is found
+            - 'v1-dependencies-{{ checksum "package.json" }}'
             - v1-dependencies-
-
-      - run: yarn install
-
+      - run: 'yarn install'
       - save_cache:
+          key: 'v1-dependencies-{{ checksum "package.json" }}'
           paths:
             - node_modules
-          key: v1-dependencies-{{ checksum "package.json" }}
-
-      # run tests!
-      - run: yarn test
+      - run: 'yarn lint'
+    working_directory: ~/repo
   snapshots:
     docker:
-      - image: circleci/node:9.11.2
-
-    working_directory: ~/repo
-
+      - image: 'circleci/node:9.11.2'
     steps:
       - checkout
-
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package.json" }}
-            # fallback to using the latest cache if no exact match is found
+            - 'v1-dependencies-{{ checksum "package.json" }}'
             - v1-dependencies-
       - run:
+          command: 'npm run test:snapshots'
           name: Snapshots
-          command: npm run snapshots
+    working_directory: ~/repo
+version: 2
 workflows:
-  version: 2
   build_and_test:
     jobs:
       - lint
+      - lighthouse
       - snapshots
+  version: 2
```

Comment:
do we need this line and line 47 as well? since yaml is indent-based, I'm afraid this is redundant? Or probably I'm missing something :)